rest/endpoints/user_recordings/profile_recordings.yml

```
@@ -776,19 +847,21 @@ paths:
                   data:
                     type: array
                     items:
-                      $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/UserRecordings/RecordingGroupEntity.yml#/RecordingGroupEntity"
+                      $ref: "../../snippets/UserRecordings/RecordingGroupEntity.yml#/RecordingGroupEntity"
                   included:
                     type: array
                     items:
-                      $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/EnrichedMetadata/SeriesEntity.yml#/SeriesEntity"
+                      anyOf:
+                        - $ref: "../../snippets/EnrichedMetadata/SeriesEntity.yml#/SeriesEntity"
+                        - $ref: "../../snippets/EnrichedMetadata/ChannelV4Entity.yml#/ChannelEntity"
```

I think you can reference entities from SwaggerHub directly, so we don't need to copy the definitions here.
```suggestion
                        - $ref: "https://api.swaggerhub.com/domains/24i-API/Snippets/v1.0.0#/components/schemas/ChannelResource"
```